common/ethereum/src/eip_4844.rs

```diff
@@ -0,0 +1,58 @@
+use common::types::Result;
+use common_chain_ids::EthChainId;
+use derive_more::Constructor;
+use ethereum_types::U256;
+
+use crate::EthBlock;
+
+#[derive(Clone, Constructor)]
+pub struct Eip4844 {}
+
+impl Eip4844 {
+    #[allow(dead_code)]
+    pub fn get_activation_block_number(&self, eth_chain_id: &EthChainId) -> Result<U256> {
+        match eth_chain_id {
+            EthChainId::Goerli => Ok(U256::from(10_456_115)),
+            EthChainId::Sepolia => Ok(U256::from(5_207_710)),
+            _ => Err(format!("{} does not have an `EIP4844` activation block number! ", eth_chain_id).into()),
+        }
+    }
+
+    pub fn is_active(&self, block: &EthBlock) -> Result<bool> {
+        Ok(block.blob_gas_used.is_some())
+    }
```

Comment:
There's no need for this to return a `Result`.

```suggestion
    pub fn is_active(&self, block: &EthBlock) -> bool {
        block.blob_gas_used.is_some()
    }
```

Reply:
I kept the same signature of `Eip1559::is_active`:
https://github.com/pnetwork-association/ptokens-core/blob/e05dd3bdcc608139f4ce7353e435f508ea314f4c/common/ethereum/src/eip_1559.rs#L18